portfolio/src/main/webapp/index.html

```diff
@@ -52,13 +52,20 @@ <h1>Home</h1>
         <br>
         <div id="house-container"></div>
         <br>
-        <button onclick="getComments()">Fetch test</button>
+        <select id="num-comments" onchange="getComments()">
+          <option value="5">5</option>
+          <option value="10">10</option>
+          <option value="20">20</option>
+          <option value="50">50</option>
+        </select>
+        <!-- <button onclick="deleteComments()">Delete All Comments</button> -->
```

Comment:
This is fine for for a demo project. In a real project, this probably should be protected by an authentication layer.

Reply:
Good point, thanks